apis/management.cattle.io/v3/globaldns_types.go

```diff
@@ -52,13 +52,18 @@ type GlobalDNSProviderSpec struct {
 }
 
 type Route53ProviderConfig struct {
-	AccessKey string `json:"accessKey" norman:"notnullable,required,minLength=1"`
-	SecretKey string `json:"secretKey" norman:"notnullable,required,minLength=1,type=password"`
+	AccessKey       string `json:"accessKey" norman:"notnullable,required,minLength=1"`
+	SecretKey       string `json:"secretKey" norman:"notnullable,required,minLength=1,type=password"`
+	CredentialsPath string `json:"credentialsPath" norman:"default=/.aws"`
+	RoleArn         string `json:"roleArn"`
```

Comment:
shall we add omitempty to all the new values?

Reply:
I think we dont need omitempty for the ones which have a default specified. For the property "RoleArn" I will add omitempty.